.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+name: quality checks
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - 3.7
+          - 3.8
+          - 3.9
+          - "3.10"
```

Comment:
why the explicit casting, isn't this an ordered list already??

Reply:
I'm just guessing here, but could it be that through some YAML weirdness it gets parsed as a float and interpreted as "3.1" if you don't make it an explicit string?

Reply:
true, my bad!